src/components/Utils/createOrder.js

```diff
@@ -0,0 +1,23 @@
+import { cloneDeep } from 'lodash';
+
+const createOrder = (order, mutator) => {
+  const newOrder = cloneDeep(order);
+
+  delete newOrder.assigned_to_user;
+  delete newOrder.created_by_name;
+  delete newOrder.vendor_name;
+  if (newOrder.id) {
```

Comment:
I don't think it's needed here, what I meant - to check this in some common `saveOrder` function to choose appropriate method of mutator - PUT or POST. If you have here only the POST, so no need to this check